packages/components/src/tabs/styles.ts

```diff
@@ -13,12 +13,31 @@ import { space } from '../utils/space';
 import { reduceMotion } from '../utils/reduce-motion';
 
 export const TabListWrapper = styled.div`
+	position: relative;
 	display: flex;
 	align-items: stretch;
 	flex-direction: row;
 	&[aria-orientation='vertical'] {
 		flex-direction: column;
 	}
+	&::after {
```

Comment:
This is unfortunately one of those times where we need to deal with downstream overrides just because we live in the same repo.

The Preferences modal can be shown from the dropdown menu in the top right of the block editor.

https://github.com/WordPress/gutenberg/blob/f003f28c1d1b01d45a0b3455bd90cb671b7a993f/packages/preferences/src/components/preferences-modal-tabs/style.scss#L32-L34

## Expected

(Don't mind the different accent color, that's just due to the theme.)

<img width="787" alt="Preferences panel in trunk" src="https://github.com/WordPress/gutenberg/assets/555336/ed0d9a36-fafd-46fa-b151-1ff49098d96d">

## Actual

<img width="777" alt="Preferences modal in this PR" src="https://github.com/WordPress/gutenberg/assets/555336/c6dab404-e67c-4849-9ba2-5b1813f1d65c">

Reply:
Fixed.